modules/rest/src/main/java/io/oasp/module/rest/service/impl/RestServiceExceptionFacade.java

```diff
@@ -23,9 +23,9 @@
 
 import net.sf.mmm.util.exception.api.NlsRuntimeException;
 import net.sf.mmm.util.exception.api.TechnicalErrorUserException;
+import net.sf.mmm.util.exception.api.ValidationErrorUserException;
```

Comment:
I double checked that this is the only location where the deprecation warning appears.